src/Util.hpp

```diff
@@ -33,6 +35,22 @@
 
 class Context;
 
+// Helper variadic template function to construct native paths.
+// Used like: PATH("usr", "local", "bin");
+template<typename... T>
+std::string
+PATH(T&... args)
```

Comment:
Is there a reason why the name is uppercase? maybe I'm not familiar with ccache code style but if it's a function then probably it's better to make a name more descriptive like `make_path` for example.

Additionally I think it's better to add `const` before `T` just in case.

Reply:
How strongly do you feel about this? I could make it something like `Util::make_path(...)` instead, but I thought that since we already have a `FMT(...)` helper this would be in a similar vein.

Reply:
@jrosdahl and which would you prefer? `Util::make_path(...)` would probably not be much worse.

Reply:
I agree that it's for @jrosdahl to decide, depending if there is separation of naming style for macros and functions etc.

Reply:
`Util::make_path` sounds good to me.

Reply:
I've made this change.